oci/client/internal/fs/fs_test.go

```diff
@@ -41,7 +41,6 @@ func TestMain(m *testing.M) {
 	}
 
 	os.MkdirAll("testdata/symlinks", 0o755)
-	defer os.RemoveAll("testdata/symlinks")
```

Comment:
Are OCI tests broken in main? If so please move this to a different PR as it's not related to client.delete.

Reply:
They aren't broken per se. The defer function never runs because of os.Exit() so the created directory doesn't get removed.

But you are right. I would move it to a different pr